lib/rubocop/cop/bugcrowd/add_index_non_concurrently.rb

```diff
@@ -28,7 +28,11 @@ class AddIndexNonConcurrently < Cop
         PATTERN
 
         def_node_matcher :add_index_with_concurrent?, <<~PATTERN
-          (send nil? :add_index _ _ <(sym :algorithm) (sym :concurrently) ...>)
```

Comment:
This matcher was always gonna return false (in most cases), therefore anything that was `add_index?` true was also an offense...i.e. all `add_index`s were a violation